Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -3498,6 +3508,29 @@ instance (priority := 100) Subgroup.normal_subgroupOf {H N : Subgroup G} [N.Norm
 #align subgroup.normal_subgroup_of Subgroup.normal_subgroupOf
 #align add_subgroup.normal_add_subgroup_of AddSubgroup.normal_addSubgroupOf
 
+theorem Subgroup.map_normalClosure_equiv_eq_normalClosure_image {s : Set G} (f : G ≃* N) :
+    (normalClosure s).map f = normalClosure (f '' s) := by
```

Comment:
I think this would make more sense as
```suggestion
theorem Subgroup.map_normalClosure {s : Set G} (f : G ≃* N) :
    (normalClosure s).map f = normalClosure (f '' s) := by
```
as this matches `map_closure`